docs/project-setup.md

```diff
@@ -0,0 +1,79 @@
+# Setting up a new project
+This document covers the basic project management tools and procedures recommended for Special Interest Group developing an implementation of the OpenTelemetry Specification for a given programming language.
+
+A Special Interest Group focused on implemeting OpenTelemetry in a programming language is reffered to as a **project** for the remainder of this document.
+
+If you are interested in starting a new project, please do the following
+* [create an issue](github.com/open-telemetry/community/issues) proposing the new project.
+* [Configure the new repository](docs/how-to-configure-new-repository.md).
+* Add the milestones listed below to the backlog.
+
+
+# Roadmap to Convergence
+For each new language, we want to quickly achieve parity with existing OpenTracing and OpenCensus implementations.
+
+[Merging OpenTracing and OpenCensus: A Roadmap to Convergence](https://medium.com/opentracing/a-roadmap-to-convergence-b074e5815289) defines the initial goals and timelines for OpenTelemetry.
+
+## OpenTelemetry Timeline
+**4/24** The reference candidate is presented for review.
+
+**5/8** Teams form, and work begins in all languages.
+
+**9/6** C#, Golang, Java, NodeJS, and Python reach parity with existing projects.
+
+**11/6** OpenTracing and OpenCensus projects are officially sunset. 
```

Comment:
do we need to track this milestone as a part of this project? I'm happy to, but perhaps a better place may be OpenTracing repositories

Reply:
Yes, we should create something in the backlog for each OT/OC project. Still, I think it would be good to keep a tracking issue here as well, so that the overall process can be linked together.